pkg/status/render_test.go

```diff
@@ -0,0 +1,3221 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+//go:build !serverless && otlp
+// +build !serverless,otlp
+
+package status
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFormatStatus(t *testing.T) {
+	time.Local = time.UTC
+	t.Run("invalid JSON", func(t *testing.T) {
+		actual, err := FormatStatus([]byte(`-`))
+		assert.Nil(t, err)
+		want := `
+====================
+Status render errors
+====================
+  invalid character ' ' in numeric literal
+
+
+`
+		assert.Equal(t, want, actual)
+	})
+
+	t.Run("success", func(t *testing.T) {
+		actual, err := FormatStatus([]byte(agentStatusJSON))
+		assert.Nil(t, err)
+		want := `
```

Comment:
What happens if something is added to the status? Does this test still pass? 
I think for `success`, you can just check that the follow lines are not present.
```
====================
Status render errors
====================
```